dags/process_elife_zip_dag.py

```diff
@@ -197,14 +197,28 @@ def wrap_article_in_libero_xml_and_send_to_service(**context) -> None:
         root
     )
 
+    return etree.tostring(xml, xml_declaration=True, encoding='UTF-8')
+
+
+def send_article_to_content_service(**context) -> None:
+    libero_xml = get_article_from_previous_task(context)
+
+    # get article id
+    xpath = '//libero:item/jats:article/jats:front/jats:article-meta/jats:article-id[@pub-id-type="publisher-id"]'
```

Comment:
is this supremely complex xpath only used here? sounds like a constant to put at the top of the module for reuse?

Reply:
Correct, only used here in this context.